packages/apps/patterns/react-appkit/src/components/StatusIndicator/StatusIndicator.tsx

```diff
@@ -18,8 +18,8 @@ export const StatusIndicator = () => {
   const client = useClient();
   const handleStateToggle = async () => {
     void (state === ConnectionState.ONLINE
-      ? client.mesh.setConnectionState(ConnectionState.OFFLINE)
-      : client.mesh.setConnectionState(ConnectionState.ONLINE));
+      ? client.setConnectionState(ConnectionState.OFFLINE)
```

Comment:
i don't think i was ready to do mesh yet until we have another abstraction (e.g., client.network)
can this be easily done later since it isn't a core part of the API?

Reply:
so you're saying you'd like to leave `mesh` as a property of `Client` for the time being until we flesh that out a bit more?